packages/venia-concept/webpack.config.js

```diff
@@ -72,7 +72,7 @@ module.exports = async function(env = {}) {
                     ]
                 },
                 {
-                    include: [themePaths.src, /peregrine\/src\//],
+                    include: [themePaths.src, /peregrine/],
```

Comment:
We don't need to run all of peregrine through babel, right?

```suggestion
                    include: [themePaths.src, /peregrine\/lib\//],
```

Reply:
That's a filter expression for files that get required during Webpack compilation. It's not a command to include that whole directory, it's just to say "if we load a module whose path matches this regex, then run it through this rule chain".